docs/groovy-script/mods/abyssaltweaker/fuel.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Fuel"
+titleTemplate: "Abyssal Tweaker | CleanroomMC"
+description: "Allows configuring the items that can be used as Transmutator and/or Crystallizer fuels."
+source_code_link: "https://github.com/CleanroomMC/GroovyScript/blob/master/src/main/java/com/teamdimensional/abyssaltweaker/compat/groovyscript/Fuel.java"
```

Comment:
this link is incorrect, but the other ones are correct - whats going on here?

Reply:
Uh oh

Reply:
I feel like this is not on me because [linkGenerator is properly set](https://github.com/TeamDimensional/AbyssalTweaker/blob/master/src/main/java/com/teamdimensional/abyssaltweaker/compat/groovyscript/Fuel.java#L20). Likely it doesn't work properly with NamedRegistry or something.